vllm/model_executor/models/internvl.py

```diff
@@ -583,16 +590,25 @@ def forward(
                 inputs_embeds = merge_multimodal_embeddings(
                     input_ids, inputs_embeds, vision_embeddings,
                     self.img_context_token_id)
+                visual_token_mask = (
+                    input_ids == self.img_context_token_id).reshape(-1, 1)
```

Comment:
Hmm, this would result in unnecessary overhead when not using Mono-InternVL. Can we run this conditionally?